### PR TITLE
Devex html pdf templates

### PR DESCRIPTION
**Pull request by matthopson**

### CODE REVIEW

File: shared/src/business/utilities/generateHTMLTemplateForPDF.js [hunk 1]
```diff
@@ -0,0 +1,326 @@
...
+    name,
+    newData,
+    oldData,
+  } = content;
+
+  let oldAddress = '';
+  let newAddress = '';
+
+  if (documentTitle === 'Notice of Change of Telephone Number') {
+    oldAddress = `<div>${oldData.phone}</dvi>`;
```
Comment (kkoskelin) [inline shared/src/business/utilities/generateHTMLTemplateForPDF.js]: oops `</dvi>`

File: shared/src/business/utilities/generateHTMLTemplateForPDF.js [hunk 2]
```diff
@@ -0,0 +1,326 @@
...
+    if (oldData.country) {
+      oldAddress += `<div>${oldData.country}</div>`;
+    }
+
+    if (newData.country) {
+      newAddress += `<div>${newData.country}</div>`;
+    }
+
+    if (documentTitle === 'Notice of Change of Address and Telephone Number') {
+      oldAddress += `<div style="margin-top:8px;">${oldData.phone}</dvi>`;
```
Comment (kkoskelin) [inline shared/src/business/utilities/generateHTMLTemplateForPDF.js]: another pair of `</dvi>`

File: shared/src/business/utilities/generateHTMLTemplateForPDF.test.js [hunk 3]
```diff
@@ -0,0 +1,324 @@
...
+    captionPostfix: 'Test Caption Postfix',
+    docketNumberWithSuffix: '123-45S',
+    docketRecord: '<table id="test-docket-record"></table>',
+    partyInfo: '<table id="test-party-info"></table>',
+  };
+
+  it('Returns HTML with the given content', () => {
+    const result = generatePrintableDocketRecordTemplate(content);
+
+    expect(result.indexOf('Test Case Caption')).toBeGreaterThan(-1);
```
Comment (kkoskelin) [inline shared/src/business/utilities/generateHTMLTemplateForPDF.test.js:314]: I believe most of the `.indexOf(string) > -1)` comparisons can generally be replaced with `.contains(string)`

**Review by kkoskelin — CHANGES_REQUESTED** [review]
Just a few typos to address to ensure valid HTML.
Eric mentioned to me at one time that perhaps we could consider using PUG templates, which I think would be a good idea.  That's not a directive for this PR, but just an observation. :)

File: shared/src/business/utilities/generateHTMLTemplateForPDF.test.js (hunk 3 above)
Comment (matthopson) [reply shared/src/business/utilities/generateHTMLTemplateForPDF.test.js:314]: Screen Shot 2019-08-28 at 7 25 18 AM
I don't think .contains is the droids, haha.

**Comment by matthopson** [comment]
@kkoskelin I'm reverting the change to `includes` (assume what you meant by `contains`) since it's technically not supported in IE11, and polyfills just use indexOf anyway.

File: shared/src/business/utilities/generateHTMLTemplateForPDF.test.js (hunk 3 above)
Comment (kkoskelin) [reply shared/src/business/utilities/generateHTMLTemplateForPDF.test.js:314]: You're right, I meant `.includes`, d'oh NBD

**Review by kkoskelin — APPROVED** [review]

**Pull request merged**